config/locales/views/imports/ru.yml

```
@@ -0,0 +1,437 @@
+      invalid_file_type: Неверный тип файла. Пожалуйста, загрузите CSV файл.
+      csv_uploaded: CSV успешно загружен.
+      ndjson_uploaded: NDJSON файл успешно загружен.
+      pdf_too_large: PDF файл слишком большой. Максимальный размер %{max_size}MB.
+      pdf_processing: Ваш PDF обрабатывается. Вы получите электронное письмо, когда анализ будет завершён.
+      invalid_pdf: Загруженный файл не является действительным PDF.
+      document_too_large: Файл документа слишком большой. Максимальный размер %{max_size}MB.
+      invalid_document_file_type: Неверный тип файла документа для активного хранилища векторов.
+      document_uploaded: Документ успешно загружен.
+      document_upload_failed: Не удалось загрузить документ в хранилище векторов. Пожалуйста, попробуйте снова.
+      invalid_ndjson_file_type: Неверный тип или формат файла. Пожалуйста, загрузите действительный .ndjson или .json файл экспорта.
+      ndjson_uploaded: NDJSON файл успешно загружен.
```

YAMLlint (1.38.0)

[error] duplication of key "ndjson_uploaded" in mapping

(key-duplicates)